event_mail/hooks.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2019 David Vidal <david.vidal@tecnativa.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+
+def pre_init_hook(cr):
+    """Create company event_mail_template_id column in advance"""
```

Comment:
Why are you doing this? Don't document the "what", which can be seen reading the code, but the "why"

Reply:
Right. It's because it's being called in a field default value, so it's needed in advance

Reply:
Do you mean for not having any default value in it?

Reply:
No, because the column doesn't exist yet in the model res.company.

Reply:
Uhm, curious. I would say that this shouldn't happen, but it seems Python order sometimes matters... Maybe you can change the import order for avoiding this hook.

Reply:
That was it :)